gensim/models/wrappers/dtmmodel.py

```diff
@@ -6,16 +6,17 @@
 # Based on Copyright (C) 2014 Radim Rehurek <radimrehurek@seznam.cz>
 
 
-"""
-Python wrapper for Dynamic Topic Models (DTM) and the Document Influence Model (DIM)  [1].
+"""Python wrapper for Dynamic Topic Models (DTM) and the Document Influence Model (DIM)  [1].
 
 This module allows for DTM and DIM model estimation from a training corpus.
 
-Example:
-
+Examples
+--------
 >>> model = gensim.models.wrappers.DtmModel('dtm-win64.exe', my_corpus, my_timeslices,
```

Comment:
Please add imports to examples (this impossible to make it really executable, but this can be very near to executable variant (i.e. all needed imports, define all variables, etc), here and everywhere.